### PR TITLE
update image to use prow_config if exists

### PR DESCRIPTION
**Pull request by lluunn**
depending on https://github.com/kubeflow/testing/pull/39

After this change, we will update the testing image for kubeflow/kubeflow to:
 - use prow_config.yaml if exists
 - else, use legacy workflow

This will be backward compatible, and we can check in the prow_config.yaml later.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/275)

### CODE REVIEW

**Comment by lluunn** [comment]
related #276

File: testing/run.sh [hunk 1]
```diff
@@ -26,4 +26,6 @@ python -m kubeflow.testing.run_e2e_workflow \
 	--cluster=kubeflow-testing \
 	--bucket=kubernetes-jenkins \
 	--component=workflows \
-	--app_dir=/src/kubeflow/kubeflow/testing/workflows
+	--app_dir=/src/kubeflow/kubeflow/testing/workflows \
```
Comment (jlewi) [inline testing/run.sh]: Just check if prow_config.yaml exists and if it doesn't invoke the current command and if it does invoke the new command.

Then you can rebuild the image and the image will be backwards compatible. Then you don't depend on any changes  to kubeflow/testing.

**Comment by lluunn** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[testing/run.sh, line 29 at r1](https://beta.reviewable.io/reviews/kubeflow/kubeflow/275#-L5u7LDIJrWQxzO-6wnl-r1-29:-L5uYToh8LW893U9jAhl:bb373kx) ([raw file](https://github.com/kubeflow/kubeflow/blob/4cad818d0f3fd2f026d8c565c3205d167ffc3c52/testing/run.sh#L29)):*
<details><summary><i>Previously, jlewi (Jeremy Lewi) wrote…</i></summary><blockquote>

Just check if prow_config.yaml exists and if it doesn't invoke the current command and if it does invoke the new command.

Then you can rebuild the image and the image will be backwards compatible. Then you don't depend on any changes  to kubeflow/testing.
</blockquote></details>

Done. I will close https://github.com/kubeflow/testing/pull/39

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/kubeflow/kubeflow/275)*

**Comment by lluunn** [comment]
@jlewi PTAL. If it looks good, I will go ahead push the updated image.

**Comment by jlewi** [comment]
Looks good. Please build/push a new image and then rerun the presubmits to test its still passing.

**Comment by lluunn** [comment]
@jlewi the image pushed, and presubmit test passed. PTAL

**Pull request merged**